src/jsonData/newEvaluation.json

```diff
@@ -0,0 +1,492 @@
+{  
+    "technicalLevelElements":{  
+       "title":"Technical Level",
+       "headers":[  
+          "Trainee",
+          "Junior",
+          "Middle",
+          "Senior"
+       ],
+       "columnData":[  
+          {  
+             "inputLevels":[  
+                "trainee"
+             ]
+          },
+          {  
+             "inputLevels":[  
+                "junior1",
+                "junior2",
+                "junior3"
+             ]
+          },
+          {  
+             "inputLevels":[  
+                "middle1",
+                "middle2",
+                "middle3"
+             ]
+          },
+          {  
+             "inputLevels":[  
+                "senior1",
+                "senior2",
+                "senior3"
+             ]
+          }
+       ]
+    },
+    "textareaSectionElements":[  
+       {  
+          "header":"Should the candidate be hired?",
+          "placeholder":"The type of project that is suitable for the candidate. &#10; Is guidance required for the candidate"
+       },
+       {  
+          "header":"General Impression",
+          "placeholder":"*required"
+       },
+       {  
+          "header":"Workflow, Leadership &amp; Soft Skills",
+          "placeholder":"Describe the environment in which the candidate works. &#10; Describe any guidance or management experience."
+       }
+    ],
+    "fieldsetSectionElements":{  
+       "data":[  
+          {  
+             "legend":"OOP, Design Patterns",
+             "items":[  
+                {  
+                   "label":"Classes",
+                   "name":"classes",
+                   "option":[  
```

Comment:
All of these `option`s are repetitive, so we could go one step further and abstract them away into a separate, generic, reusable property. That way, your config would be DRY, alongside your code.